### PR TITLE
New role for installing and configuring haproxy

### PR DESCRIPTION
**Pull request by carsongee**
This can be useful for managing rabbitmq clusters and possibly
as a replacement for ELB.

### CODE REVIEW

File: playbooks/roles/haproxy/files/haproxy.logrotate [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/var/log/haproxy/*.log {
```
Comment (feanil) [inline playbooks/roles/haproxy/files/haproxy.logrotate]: Can you use the common log dir here instead of an explicit path?

Comment (carsongee) [inline playbooks/roles/haproxy/files/haproxy.logrotate]: good catch

**Comment by carsongee** [comment]
I switched it to use common log dir, should also make a specific logdir, or is it ok to leave haproxy as the static log subfolder?

**Comment by feanil** [comment]
looks good to me as is.  :+1:

**Pull request merged**